docs/source/algorithms/IntegratePeaksShoeboxTOF-v1.rst

```diff
@@ -0,0 +1,86 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This is an algorithm to integrate single-crystal Bragg peaks in a :ref:`MatrixWorkspace <MatrixWorkspace>` by
```

Comment:
I think the explanation of the algorithm is fine, but I'm missing some context regarding how the integration is perform, or what's the advantage/use case with respect to IntegrateSkew? Perhaps adding a reference of neutron scattering literature where this procedure is explained or used may be useful?

Reply:
Good idea, I'll try and find one, it's a new algorithm for dynamic shoebox on TOF Laue, but there will be similar algorithms out there I can find - at least for x-ray.